src/Components/Aspire.Microsoft.Azure.Cosmos/AzureCosmosDBSettings.cs

```diff
@@ -37,5 +37,10 @@ public sealed class AzureCosmosDBSettings
     /// Gets or sets the credential used to authenticate to the Azure Cosmos DB endpoint.
     /// </summary>
     public TokenCredential? Credential { get; set; }
+
+    /// <summary>
+    /// Controls whether the Cosmos DB emulator certificate is ignored when its use is detected.
+    /// </summary>
+    public bool IgnoreEmulatorCertificate { get; set; }
```

Comment:
Once we update to a version that has https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4251, would we remove this public API?

Reply:
Oh I wasn't aware of that PR. Thanks for pointing it out. Short answer, yes, if they add support for doing this via connection string then we should probably remove this flag.